flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -471,6 +471,11 @@ function wait_job_terminal_state {
   done
 }
 
+function stop_with_savepoint {
+  echo "INSIDE: Stopping job $1 with a savepoint @ $2."
```

Comment:
Is the printing meant to stay here?

Reply:
Nope, I will remove it. It stayed after debugging.